src/elements/clock/clock.ts

```diff
@@ -236,13 +236,6 @@ export class SbbClockElement extends LitElement {
 
     if (this.now) {
       this._setHandsStartingPosition();
-
```

Comment:
Yes, but I could not reproduce the snapshot problem in the first place.
Anyway, those lines are redundant since the same classes are applied by the `_setHandsStartingPosition();` method.
The problem could be caused by the `setTimeout` since it lets the animation running for that frame

Reply:
I re-run the visual tests 3 times and the clock appears stable. (I have failures but might be because the baseline is faulted)

Reply:
Does maybe @DavideMininni-Fincons know more about the history of this piece of code? Or was it myself introducing it? :D I don't remember

Reply:
found it: 
https://github.com/sbb-design-systems/lyne-components/pull/2674/commits/35b51871929e161a7c00665b1870c4d42d0f639e#diff-35688b834881912fec6d156fdf8042c6bb958b9e0180632fee8d1a0a3bdaacf0